addons/binding/org.openhab.binding.jenocean/README.md

```diff
@@ -0,0 +1,88 @@
+# EnOcean Binding
+
+This binding enables support for receiving messages from EnOcean rocker switches.
+
+
+## Supported Things
+
+Supports only EnOcean rocker wall switches with EEP F6-02-02.
+Support for other devices is planned but not supported as of yet. The underlying library used for communication does support both sensors and actors though.
+
+## Discovery
+
+This binding does not support device discovery.
```

Comment:
... do you see a chance to implement this with the EnJ library? Imho that would be a pretty neat feature - if telegrams are received from some unknown device, it could be automatically put in the inbox. Is there a chance to determine the device type in such a moment?

Reply:
It might be. I have thought about it myself before, but this was firmly in futureland. If it is up to me, this would be placed on a nice-to-have list and perhaps implemented after an initial release. I am sure that the users would much prefer to have more supported profiles than this feature. 

In order to determine if this is possible, I would need to find out if devices other than the rocker switch share an EEP with it.
Afaik the telegrams contain this EEP identifier.